torch/_torch_docs.py

```diff
@@ -210,6 +210,75 @@ def merge_dicts(*dicts):
             [ -8.9902,  -8.3667,  -7.3925,  -7.6147]])
 """.format(**common_args))
 
+add_docstr(torch.sub,
```

Comment:
It would also be nice if this could be placed in alphabetical order within the file.